src/datetime-picker.component.ts

```diff
@@ -140,6 +140,15 @@ declare var moment: any;
   `,
   styles: [
     `
+* {
```

Comment:
Ah, didn't pay attention to that. Then the star-rule is really bad. I will fix that and update the pull request.

I cannot really say why content-box was being used. My guess is that it is simply used because it is the implicit choice when not specifying a box sizing model. 

In any case I'll fix the crazy * selector first. Redoing all pixel sizes to border-box is a bit more work.